openapi/horizon/bundled.yml

```diff
@@ -1163,6 +1163,145 @@ paths:
+        '403':
+          description: Transaction submission has been disabled for Horizon.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Problem'
+              examples:
+                TransactionSubmissionDisabled:
+                  summary: Transaction Submission Disabled
+                  value:
+                    type: transaction_submission_disabled
+                    title: Transaction Submission Disabled
+                    status: 403
+                    detail: Transaction submission has been disabled for Horizon. To enable it again, remove env variable DISABLE_TX_SUB.
+                    extras:
+                      envelope_xdr: ''
+        '409':
+          description: Transaction is a duplicate of a previously submitted transaction.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AsyncTransactionSubmissionResponse'
+              examples:
+                TransactionDuplicate:
+                  summary: Transaction Duplicate
+                  value:
+                    tx_status: DUPLICATE
+                    hash: 6cbb7f714bd08cea7c30cab7818a35c510cbbfc0a6aa06172a1e94146ecf0165
+        '500':
+          description: Transaction submission failure, exception or invalid status from core.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Problem'
+              examples:
+                TransactionFailed:
+                  summary: Transaction Submission Failed
+                  value:
+                    type: transaction_submission_failed
+                    title: Transaction Submission Failed
+                    status: 500
+                    detail: 'Could not submit transaction to stellar-core. The `extras.error` field on this response contains further details. Descriptions of each code can be found at: https://developers.stellar.org/api/errors/http-status-codes/horizon-specific/transaction-submission-async/transaction_submission_failed'
+                    extras:
+                      envelope_xdr: ''
+                      error: Error details here
+                TransactionException:
+                  summary: Transaction Submission Exception
+                  value:
+                    type: transaction_submission_exception
+                    title: Transaction Submission Exception
+                    status: 500
+                    detail: 'Received exception from stellar-core. The `extras.error` field on this response contains further details. Descriptions of each code can be found at: https://developers.stellar.org/api/errors/http-status-codes/horizon-specific/transaction-submission-async/transaction_submission_exception'
+                    extras:
+                      envelope_xdr: ''
+                      error: Exception details here
```

Comment:
I'm not familiar with this response case. When would this occur?

Reply:
I think this is probably a failure in the Soroban runtime, like a server 500 error. We can resolve.